app/assets/javascripts/controllers/container_dashboard/util.js

```diff
@@ -0,0 +1 @@
+angular.module('miq.util', []);
```

Comment:
why is this change needed - if I understand the file was just moved and renamed?